src/erlcloud_sm.erl

```diff
@@ -389,3 +443,14 @@ create_secret_payload(SecretName, Opts) ->
         [{<<"Name">>, SecretName} | Opts]),
     Json.
 
+put_secret_payload(SecretId, Opts) ->
+    Json = lists:map(
+        fun
+            ({client_request_token, Val}) -> {<<"ClientRequestToken">>, Val};
+            ({secret_binary, Val}) -> {<<"SecretBinary">>, Val};
+            ({secret_string, Val}) -> {<<"SecretString">>, Val};
+            ({version_stages, Val}) -> {<<"VersionStages">>, Val};
+            (Other) -> Other
```

Comment:
is it a good idea to leak open interface here? maybe at least limits to `{binary(), term()}`